portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/bean/UserListBean.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.is.portal.user.client.api.bean;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * User List Entry
+ */
+public class UserListBean {
+
+    private String userId;
+
+    private String userUniqueId;
+
+    private String domainName;
+
+    private String state;
+
+    private List<String> groups;
```

Comment:
Useful to add a comment on what are kept in this list. Whether it's group names claim, groupunique ID etc..